source/documentation/getting-started.html.md

```diff
@@ -1052,6 +1052,113 @@ Result:
 ManageIQ refreshes the details for the chosen cloud providers.
 
 
+## Containers
+
+The currently supported container provider types are:
+
+- [Kubernetes](http://kubernetes.io/)
+- [OpenShift Origin](http://www.openshift.org/)
+- [OpenShift Enterprise](https://enterprise.openshift.com/)
+- [Atomic](http://www.projectatomic.io/)
+- [Atomic Enterprise](https://access.redhat.com/products/red-hat-atomic-enterprise-platform)
+
+### Service Accounts
+
+To add a Kubernetes or OpenShift provider you must create, in those clusters, a specific `management` service account with the proper role,
+permissions, and authentication token.
+
+For more information on these topics, check out the relevant documentation:
+
+- [Kubernetes Service Accounts](https://github.com/kubernetes/kubernetes/blob/release-1.1/docs/design/service_accounts.md)
+- [Kubernetes Authentication](https://github.com/kubernetes/kubernetes/blob/release-1.1/docs/admin/authentication.md)
+- [OpenShift Service Accounts](https://docs.openshift.com/enterprise/3.0/dev_guide/service_accounts.html)
+
+To add a `management` service account in an OpenShift cluster:
+
+    $ oadm new-project management-infra --description="Management Infrastructure"
```

Comment:
I think this `oadm new-project manageiq-infra --description="ManageIQ Infrastructure" --display-name="ManageIQ Infra" --admin=system:serviceaccount:manageiq-infra:manageiq-admin` prevents the need for explicitly adding the `admin` role.